core/include/tangram/tangram.h

```diff
@@ -61,16 +61,32 @@ struct SceneUpdate {
     std::string path;
     std::string value;
     SceneUpdate(std::string p, std::string v) : path(p), value(v) {}
+    SceneUpdate() {}
 };
 
+enum Error {
+    scene_update_path_not_found,
+    scene_update_path_yaml_syntax_error,
+    scene_update_value_yaml_syntax_error,
```

Comment:
For now `no_error` wouldn't be used, but we can add it in the future if there is anything that needs to return a `no_error` signal.